bin/cheribsdtest/cheribsdtest.c

```diff
@@ -82,15 +82,7 @@
 #include <libxo/xo.h>
 
 #include "cheribsdtest.h"
-#include "cheribsdtest.h"
-
-#ifdef __mips__
-#define	TRAPNO_CHERI	(T_C2E)
-#elif defined(__riscv)
-#define	TRAPNO_CHERI	(SCAUSE_CHERI)
-#else
-#error "Unsupported architecture"
-#endif
+#include "cheribsdtest_md.h"
```

Comment:
Move the riscvreg.h include into cheribsdtest_md.h?

Reply:
done